components/scream/src/dynamics/homme/interface/homme_driver_mod.F90

```diff
@@ -77,9 +77,10 @@ end subroutine prim_init_data_structures_f90
 
   subroutine prim_copy_cxx_to_f90 (copy_phis)
     use iso_c_binding,     only: c_ptr, c_loc
-    use homme_context_mod, only: tl
-    use dimensions_mod,    only: nlevp
-    use theta_f2c_mod,     only: cxx_push_results_to_f90
+    use homme_context_mod, only: tl, elem
+    use dimensions_mod,    only: nlevp, nelemd, np
+    use kinds,             only: real_kind
+    use theta_f2c_mod,     only: cxx_push_results_to_f90, init_geopotential_c, compute_gradphis_c
```

Comment:
Namely, in this subroutine, you could add the `interface` block inside this subroutine (or at the top of the module, as you wish). That way, we don't have to modify the homme folder...